config/application.rb

```diff
@@ -74,5 +74,10 @@ class Application < Rails::Application
     # We create this in application.rb instead of an initializer so that it's
     # guaranteed to be available for other initializers to use.
     config.conjur_config = Conjur::ConjurConfig.new
+
+    puts "Loaded configuration:"
```

Comment:
Should this be using the Rails logger, rather than `puts` directly?

Reply:
The Rails logger hasn't been set yet, so I would have to set it to `Logger.new(STDOUT)` or something temporarily and then let it get over-written by the environment-specific settings. I could do that but I'm not sure there's much point.